go/util/jwt/suite_test.go

```
@@ -12,10 +12,9 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	testutilmod "github.com/cosmos/cosmos-sdk/types/module/testutil"
-
 	jwttests "pkg.akt.dev/testdata/jwt"
 
+	"pkg.akt.dev/go/sdkutil"
```

golangci-lint (2.2.2)

ST1019: package "pkg.akt.dev/go/sdkutil" is being imported more than once

(staticcheck)